plonky3/src/circuit_builder.rs

```diff
@@ -1,8 +1,8 @@
 //! A plonky3 adapter for powdr
```

Comment:
We should give a spec of how we treat public values, introducing the `Pi`s and `Ci`s if you refer to them in the implementation

Reply:
Also we should probably talk about the need to move the selectors to fixed columns for this to be sound